cmds/console.c

```diff
@@ -21,39 +21,63 @@
 
 static void cmd_consoleInfo(void)
 {
-	lib_printf("sets console to device, usage: console <major.minor>");
+	lib_printf("sets console to device, optionally adding up to %d mirror consoles, usage: console <major.minor> [<major.minor> ...]", CONSOLE_MIRRORS);
```

Comment:
An alternative approach is to support multiple console devices by allowing repeated `console` commands. Did you consider this? It avoids extensive API changes.

Reply:
Yes, I wanted to enable overwriting the default console